src/components/widgets/switch/Switch.tsx

```diff
@@ -111,25 +117,24 @@ export default function Switch<T extends boolean | string | number>({
         <div
           css={{
             position: 'relative',
-            width: 10,
-            height: 10,
+            width: knobSize,
+            height: knobSize,
             ...(switchState === 'hover'
               ? selectedOption === options[0]
                 ? {
                     borderTopRightRadius: 10,
                     borderBottomRightRadius: 10,
-                    borderTopLeftRadius: 5,
-                    borderBottomLeftRadius: 5,
+                    borderTopLeftRadius: height / 4,
+                    borderBottomLeftRadius: height / 4,
                   }
```

Comment:
I personally don't like this hover effect. To me, it looks and feels like a UI mistake because I've never experienced a switch that sort of "squares off" one side of the circle on hover. It feels instead like it's overflowing a container and getting cut off. I'd prefer the circle to be larger and to not change border radius on hover. We could instead opt to enlarge the circle ever-so-slightly on hover.

I'd be curious what others think about the actual styling. The overall implementation looks okay to me, but I've learned through my own CoreUI work that a lot of the work in CoreUI was done without any feedback on styling/implementation.  To that end, I bet we could get sufficient opinions/feedback via a Slack thread in this instance.

Reply:
I agree about the hover effect. I'd guess the idea was to make it look like the knob is "pointing" to the other side, but it does come across more like a UI mistake. I like the idea of having the knob enlarge on hover.

As for the general design of the switch, I don't know that I fully like it, myself.

Reply:
We should probably bring this to a styling group meeting before switching (harhar) everything over to this component.

Reply:
let's discuss at this week's styling meeting, if that's okay with you @chowington ? 

To clarify, do you all like how the blue gets darker on the hover? Or is the proposal to have the only effect on hover be that the size of the circle changes?

Reply:
My main issue is how the circle's radius gets squared off during hover, which makes it appear to be getting cut off due to an overflow issue. But frankly, I also don't like the aesthetic of a circular toggle switch inside a rectangular container. I prefer circular toggles in oval switches and rectangular toggles in rectangular switches.